portfolio/src/main/webapp/script-charts.js

```diff
@@ -0,0 +1,62 @@
+google.charts.load('current', {'packages':['bar', 'timeline']});
+google.charts.setOnLoadCallback(drawTimeline);
+google.charts.setOnLoadCallback(drawDonationGraph);
+google.charts.setOnLoadCallback(drawProgramGraph);
+
+/** Creates a chart and adds it to the page. */
+function drawTimeline() {
+
+    const date = new Date();
+    const today = new Date(date.getFullYear(), date.getMonth(), date.getDate());
+
+    const data = new google.visualization.DataTable();
+    data.addColumn('string', 'Order');
+    data.addColumn('string', 'Position')
+    data.addColumn('date', 'Start');
+    data.addColumn('date', 'End');
+        data.addRows([
+            ['1', 'Intern at Boston Housing Authority', new Date(2020, 2, 20), new Date(2020, 4, 18)],
+            ['1', 'Google STEP Intern', new Date(2020, 4, 18), today],
```

Comment:
'2'?

Reply:
The first numbers are to indicate what row they should be on; I wanted both internships to be on the same row